.eslintrc.js

```diff
@@ -1 +1,129 @@
-module.exports = require('./.eslintrc-node');
+module.exports = {
+  root: true,
+  extends: [
+    'eslint:recommended',
+  ],
+  plugins: [
+    "ember-internal"
+  ],
+
+  rules: {
+    'semi': 'error',
+    'no-unused-vars': 'error',
+    'no-useless-escape': 'off', // TODO: bring this back
```

Comment:
FWIW, the eslint version upgrade (from 3.x to 4.x) brings some significant changes to the `no-useless-escape` rule which flags ~ 350 problems.

I disabled the rule for now, and will create another questlike issue to fix it up and remove this line.